packages/hooks/src/useFocusTrap/useFocusTrap.stories.mdx

```diff
@@ -0,0 +1,89 @@
+import { Canvas, Meta, Story } from "@storybook/addon-docs";
+import { useState } from "react";
+import { Button } from "@jobber/components/Button";
+import * as hooks from ".";
+
+<Meta title="Hooks/useFocusTrap" />
+
+# UseFocusTrap
+
+Trap focus within a DOM node.
+
+Useful for building Modals, lightboxes, menus, bottom sheets, etc. The user will
+be able to navigate the component using `Tab` and `Shift+Tab`.
+
+```ts
+import { useFocusTrap } from "@jobber/hooks";
+```
+
+<Canvas>
+  <Story name="useFocusTrap">
+    {() => {
+      const [open, setOpen] = useState(false);
+      const modalRef = hooks.useFocusTrap(open);
+      return (
+        <>
+          <Button label="Open modal" onClick={() => setOpen(true)} />
+          {open && (
+            <div
+              ref={modalRef}
+              tabIndex={0}
+              style={{
+                position: "fixed",
+                top: 0,
+                bottom: 0,
+                right: 0,
+                left: 0,
+                zIndex: "var(--elevation-modal)",
+              }}
+            >
+              <Overlay onClick={() => setOpen(false)} />
+              <Modal>
+                <Button label="first button" />
+                <Button label="second button" />
+                <Button label="third button" />
+                <Button label="close" onClick={() => setOpen(false)} />
+              </Modal>
```

Comment:
Got confused about what this modal is, I thought it was our Atlantis modal. We shouldn't do that tho since people copy paste and they might thing they have to do that on every Atlantis Modal

I think we can simplify the example to just be the buttons. Like, don't need to position fixed the parent, literally just show the buttons on the canvas (maybe wrapped in a card?) and let people tab around and see that it's trapped within that box.

Reply:
Ooh, that makes sense! I'll change the example, it's really confusing using the same names